spec/models/video_spec.rb

```diff
@@ -37,4 +38,14 @@
       expect(Video.search_by_title("")).to eq([])
     end
   end
+
+  describe "#average_rating" do
+    it "calculates the average rating of a video (from its reviews) rounded to 1 decimal place" do
```

Comment:
I am not sure about `(from its reviews)`. It should be clear or maybe just delete `()`.